### PR TITLE
Expose Streamlit App in Documentation

### LINKED ISSUE #1090: Expose Streamlit App in Documentation
I realised that I didn't update any of the docs with my Streamlit MR. Would be good to add some links to it around the place as it's a bit buried/hidden currently.

I'll open a PR for this ☺️

### PR DESCRIPTION
**Pull request by louismagowan**
## Description

Adding references/links to the Streamlit app in documentation and READMEs.

Hope that's alright! 🙏 

## Related Issue

- [x] Closes #1090 
- [ ] Related to #

## Checklist

- [x] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [x] Added necessary documentation (docstrings and/or example notebooks)
- [x] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [x] MMM
- [ ] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--1091.org.readthedocs.build/en/1091/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,6 +187,12 @@ Not sure how to start or have questions? MMM-GPT is an AI that answers questions
 
 **[Try MMM-GPT here.](https://mmm-gpt.com/)**
 
+## Explainer App: Streamlit App of MMM Concepts
```
Comment (juanitorduz) [inline README.md:190]: I suggest we put this higher up for visibility ... maybe after 
### Essential Reading for Marketing Mix Modeling (MMM) ?

**Review by juanitorduz — CHANGES_REQUESTED** [review]
Looks great, thanks! Just a small suggestion

File: README.md (hunk 1 above)
Comment (louismagowan) [reply README.md]: Done! 😁

**Review by juanitorduz — APPROVED** [review]
Great! Thanks!

**Pull request merged**